packages/theme/src/components/scroll-area.ts

```diff
@@ -12,21 +12,21 @@ const scrollAreaScrollbarClsx = ({
   orientation = 'vertical',
 }: { className?: string; orientation?: 'horizontal' | 'vertical' } = {}) =>
   clsx(
-    'flex select-none touch-none p-0.5 bg-neutral-outline transition-colors hover:bg-neutral-outline-hover',
+    'flex select-none touch-none p-0.5 bg-background-level2 transition-colors hover:bg-background-level3',
     orientation === 'vertical' ? 'w-2' : 'h-2 flex-col',
     className,
   )
 
 const scrollAreaThumbClsx = ({ className = '' }: { className?: string } = {}) =>
   clsx(
-    'flex-1 bg-neutral-solid rounded-lg relative before:content-[""] before:absolute before:w-full before:h-full before:min-w-[44px] before:min-h-[44px] before:top-1/2 before:left-1/2 before:-translate-x-1/2 before:-translate-y-1/2',
+    'flex-1 bg-text-primary rounded-lg relative before:content-[""] before:absolute before:w-full before:h-full before:min-w-[44px] before:min-h-[44px] before:top-1/2 before:left-1/2 before:-translate-x-1/2 before:-translate-y-1/2',
     className,
   )
 
 const scrollAreaCornerClsx = ({
   className = '',
 }: { className?: string } = {}) =>
-  clsx('bg-neutral-outline-hover !bg-black', className)
+  clsx('bg-background-level3 !bg-black', className)
```

Comment:
I think for all of the case, the scrollbar should be transparent and the thumb should be opaque white for the dark. Can we make it? Current, it looks off in the dark mode.

![image](https://github.com/consolelabs/mochi-ui/assets/12707960/c9a0960f-535d-4cc2-b06a-91f6baa7be1d)